src/client/api/init.c

```diff
@@ -203,19 +203,12 @@ daos_init(void)
 	if (rc != 0)
 		D_GOTO(out_agent, rc);
 
-	/** set up client telemetry */
-	rc = dc_tm_init();
-	if (rc != 0) {
-		/* should not be fatal */
-		DL_WARN(rc, "failed to initialize client telemetry");
-	}
-
 	/** get and cache attach info of default system */
 	rc = dc_mgmt_cache_attach_info(NULL);
 	if (rc != 0)
 		D_GOTO(out_job, rc);
 
-	crt_info = daos_crt_init_opt_get(false, 1, daos_client_metric);
+	crt_info = daos_crt_init_opt_get(false, 1);
```

Comment:
minor nit, if we always pass in 1, should we should remove the extra arg?

Reply:
I think @frostedcmos made this change a little while ago... I'll let him comment on that.